### ISSUE TITLE
Heroku application error

### ISSUE THREAD

**Issue by RpRoderick** [ga-wdi-boston/full-stack-project#542]
Ok, I have been getting an error on heroku deploy.

<img width="577" alt="screen shot 2016-12-02 at 1 57 27 pm" src="https://cloud.githubusercontent.com/assets/22726092/20846365/5b7ebe84-b897-11e6-846c-c9859a80b6a1.png">

<img width="563" alt="screen shot 2016-12-02 at 1 57 40 pm" src="https://cloud.githubusercontent.com/assets/22726092/20846376/632c5f6a-b897-11e6-9886-a8bfd91715f6.png">

**Comment by RpRoderick** [comment]
I deleted my heroku apps from my repo.  Went back through the steps.  I get the same exact error every time.

**Comment by RpRoderick** [comment]
~/wdi/projects/fullstack/back (master)$ git remote -v
heroku	https://git.heroku.com/obscure-earth-97091.git (fetch)
heroku	https://git.heroku.com/obscure-earth-97091.git (push)
origin	git@github.com:RpRoderick/fullstackback.git (fetch)
origin	git@github.com:RpRoderick/fullstackback.git (push)
upstream	git@github.com:ga-wdi-boston/rails-api-template.git (fetch)
upstream	git@github.com:ga-wdi-boston/rails-api-template.git (push)

**Comment by RpRoderick** [comment]
here are my remotes.  git push heroku master runs and verifies deploy at the end of the run.

remote:        Released v5
remote:        https://obscure-earth-97091.herokuapp.com/ deployed to Heroku
remote: 
remote: Verifying deploy... done.
To https://git.heroku.com/obscure-earth-97091.git

**Comment by RpRoderick** [comment]
Note:  I have paperclip installed in my gems.  I was going to use it, but I've decided to leave it out for time reasons.  Just mentioning in case there is a conflict? It doesn't list any anywhere that I see, but that's the only build difference.

**Comment by RpRoderick** [comment]
This is the only warning that I see on build:
 ###### WARNING:
remote:        No Procfile detected, using the default web server.
remote:        We recommend explicitly declaring how to boot your server process via a Procfile.
remote:        https://devcenter.heroku.com/articles/ruby-default-web-server

**Comment by RpRoderick** [comment]
I run heroku logs and get this:

:56:28.257105+00:00 heroku[web.1]: State changed from starting to crashed
2016-12-02T18:56:29.359251+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/" host=obscure-earth-97091.herokuapp.com request_id=9059617f-b949-445b-a454-fc35b3f3a61b fwd="144.121.82.194" dyno= connect= service= status=503 bytes=
2016-12-02T18:56:29.466749+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/favicon.ico" host=obscure-earth-97091.herokuapp.com request_id=093ea6ca-fc6d-4093-93fb-c70b7adf4001 fwd="144.121.82.194" dyno= connect= service= status=503 bytes=
~/wdi/projects/fullstack/back (master)$

**Comment by RpRoderick** [comment]
and this, within the logs:

2016-12-02T18:56:28.164429+00:00 app[web.1]: /app/app/controllers/not_images_controller.rb:1:in `<top (required)>': superclass mismatch for class ImagesController (TypeError)

**Comment by gaand** [comment]
Does your API run locally?  File names and class names have explicit relationships in rails.

**Comment by RpRoderick** [comment]
Yes, runs locally just fine.  I believe that I just fixed it.  I'll explain below.

**Comment by RpRoderick** [comment]
So, I had a controller called images controller.  I made a new one (I wanted to save the code I had written on the side, in case the new approach that I was taking failed miserably.)  In doing this, I renamed my old imagecontroller = not_image_controller and left it within my controllers and kind of just forgot it was there.  On build, it associated the file name with the file class and threw an error (not entirely sure why?  because of the statement that you made above? I'd be curious to know the detailed answer).  So, in following my logs back, I found the spot that it threw the error:

2016-12-02T18:56:28.164429+00:00 app[web.1]: /app/app/controllers/not_images_controller.rb:1:in `<top (required)>': superclass mismatch for class ImagesController (TypeError)

and removed the file (which I should have removed before).  I then repushed heroku.  And it works!  So, follow your logs, it helps.
